spotifycli/spotifycli.py

```diff
@@ -22,42 +28,57 @@ def main():
     global client
     args = add_arguments()
     client = args.client
-    if args.version:
-        show_version()
-    elif args.status:
-        show_status()
-    elif args.statusshort:
-        show_status_short()
-    elif args.statusposition:
-        show_status_position()
-    elif args.song:
-        show_song()
-    elif args.songshort:
-        show_song_short()
-    elif args.artist:
-        show_artist()
-    elif args.artistshort:
-        show_artist_short()
-    elif args.album:
-        show_album()
-    elif args.position:
-        show_position()
-    elif args.playbackstatus:
-        show_playback_status()
-    elif args.lyrics:
-        show_lyrics()
-    elif args.arturl:
-        show_art_url()
-    elif args.play:
-        perform_spotify_action("Play")
-    elif args.pause:
-        perform_spotify_action("Pause")
-    elif args.playpause:
-        perform_spotify_action("PlayPause")
-    elif args.next:
-        perform_spotify_action("Next")
-    elif args.prev:
-        perform_spotify_action("Previous")
+    success = True
```

Comment:
The new structure would be that worker functions would either:
- run successfully
  - success flag will stay True, and the main() function will return 0
- encounter a known error condition, handle things gracefully, and throw a SpotifyCLIException that we can handle here
  - A nice error message will be shown to the user, the success flag will be set to false, and the main function will return 1
- encounter an unknown/unhandled exception and let the exception bubble up here to this `try` anyway
  - A standard Python exception message will be reported to the user, including the traceback, the success flag will be set to false, and the main function will return 1